cpu/stm32/include/periph/cpu_spi.h

```diff
@@ -48,6 +48,21 @@ extern "C" {
  */
 #define SPI_HWCS(x)         (SPI_HWCS_MASK | x)
 
+/**
+ * @brief   Define value for unused CS line
+ */
+#define SPI_CS_UNDEF    (GPIO_UNDEF)
```

Comment:
```suggestion
#define SPI_CS_UNDEF        (GPIO_UNDEF)
```
At will, just to be aligned with the definition above.